vispy/app/tests/test_backends.py

```diff
@@ -10,12 +10,11 @@
 
 from inspect import getargspec
 
-import vispy
-from vispy import keys
-from vispy.testing import (requires_application, assert_in, run_tests_if_main,
+from ... import app, keys
+from ...testing import (requires_application, assert_in, run_tests_if_main,
                            assert_raises)
-from vispy.app import use_app, Application
-from vispy.app.backends import _template
+from ...app import use_app, Application
+from ...app.backends import _template
```

Comment:
These on the other hand *should* be relative if possible, so these changes are worthwhile